app/views/static/alert-overview-popover.html.haml

```diff
@@ -0,0 +1,10 @@
+%div
+  .tip-title
+    {{item.name}}
+  .tip-row{"ng-repeat" => "alert in vm.hoverAlerts track by $index"}
+    .tip-col.host-col{"ng-if" => "$index < 5"}
```

Comment:
The design is to show only the first 5 alerts in the tooltip so as not to have a huge tooltip displayed. I could set the number in the controller if you think that is better.